src/stm32_platform.cpp

```diff
@@ -32,16 +32,21 @@ void Stm32Platform::restart()
 
 uint8_t * Stm32Platform::getEepromBuffer(uint16_t size)
 {
-    if (size > E2END + 1)
```

Comment:
Strictly speaking this change should be in a different PR. But since it's only a small change its ok.

Reply:
That's not a change. It must have happened as as re-sync a branch with thelsing:master.  Sorry for the slight mess.

The code is identical. 

master: https://github.com/thelsing/knx/blob/master/src/stm32_platform.cpp
patch: https://github.com/rueckix/knx/blob/patch-progbutton/src/stm32_platform.cpp